### ISSUE TITLE
It is not possible to create XRechnung in version 2.2.0

### ISSUE THREAD

**Issue by aschliewert** [stephanstapel/ZUGFeRD-csharp#180]
Hi!
From August 1st, 2022, XRechnung version **2.2.0** will be mandatory. 
Will there be an update for this?

**Comment by aschliewert** [comment]
That sounds great! 
Does it mean that the version is selected based on the date? 
If that's the case, will there still be an option to specify the version manually?

**Comment by stephanstapel** [comment]
I don't think so. From August 1st, the new format needs to be generated anyway.

**Comment by aschliewert** [comment]
Ok perfect, thank you for the information.
I'll try the update as soon as it's available!
And thank you for the quick reply!

**Comment by stephanstapel** [comment]
Done in https://github.com/stephanstapel/ZUGFeRD-csharp/commit/4cb94ddb7093c7b8b14fc52b10ff616b2ba71495
